js/bytex.js

```diff
@@ -1123,35 +1219,20 @@ module.exports = class bytex extends Exchange {
             'currency': currency['code'],
             'address': address,
             'tag': tag,
-            'network': this.safeNetwork (network),
```

Comment:
this needs to be safeNetwork, so that we return the unified values for the network ids instead of the exchange specific ones